src/utils/console-extensions.js

```diff
@@ -0,0 +1,16 @@
+const { red, green, yellow } = require('chalk')
+console.red =
```

Comment:
Why not use `src/utils/logging.js` instead of creating a duplicate?

Reply:
Done..